upcoming-release-notes/802.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Features
+authors: albertogasparin
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏I don't think our changelog automation will work if this is not an array.
```suggestion
authors: [albertogasparin]
```